Frontend/src/Components/PPC_MPPT/PPC_CELL.js

```diff
@@ -13,13 +13,13 @@ export default function PPC_CELL(props) {
     return (
         <Center
             w='47%'
-            bg={props.boolean ? greenBgCol : redBgCol}
+            bg={props.isGreen ? greenBgCol : redBgCol}
```

Comment:
what does isGreen represent? Is this just picking the color green, or is it within bounds or whats the thing here?